src/main/resources/db/migration/ledgersyncagg/V1_4_1__add-indexes.sql

```diff
@@ -1,21 +1,9 @@
-CREATE INDEX IF NOT EXISTS address_tx_amount_unit_block_time2_idx
-    ON address_tx_amount (unit, to_timestamp(block_time));
 CREATE INDEX IF NOT EXISTS address_tx_amount_stake_address_idx
     ON address_tx_amount (stake_address);
 CREATE INDEX IF NOT EXISTS address_tx_amount_epoch_idx
     ON address_tx_amount (epoch);
-CREATE INDEX IF NOT EXISTS address_tx_amount_unit_quantity_block_time_idx
-    ON address_tx_amount (unit, quantity, block_time);
-CREATE INDEX IF NOT EXISTS address_tx_amount_unit_tx_hash_block_time_idx
-    ON address_tx_amount (unit, tx_hash, block_time);
-CREATE INDEX IF NOT EXISTS address_tx_amount_address_tx_hash_block_time_idx
-    ON address_tx_amount (address, tx_hash, block_time);
-CREATE INDEX IF NOT EXISTS address_tx_amount_stake_address_tx_hash_block_time_idx
-    ON address_tx_amount (stake_address, tx_hash, block_time);
-CREATE INDEX IF NOT EXISTS address_tx_amount_stake_address_unit_block_time_idx
-    ON address_tx_amount (stake_address, unit, block_time);
-CREATE INDEX IF NOT EXISTS address_tx_amount_slot_idx
-    ON address_tx_amount (slot);
+CREATE INDEX IF NOT EXISTS address_tx_amount_block_time_idx
+    ON address_tx_amount (block_time);
```

Comment:
Is block_time needed here? We wanted to build everything based on slot

Reply:
as discussed on Slack: https://app.slack.com/client/T015QBMLJHZ/C07BPJUU7A9
Some aggregate MVs are based on the block_time column to filter records within a specified time frame (e.g., 3 months ago). We cannot convert block_time to slot in SQL, therefore, these MVs cannot be based on the slot column.